core/src/main/java/org/jdbi/v3/core/Jdbi.java

```diff
@@ -73,6 +73,13 @@ private Jdbi(ConnectionFactory connectionFactory)
     {
         Objects.requireNonNull(connectionFactory, "null connectionFactory");
         this.connectionFactory = connectionFactory;
+
+        baseConfiguration();
+    }
+
+    private void baseConfiguration()
+    {
+        configure(SqlArrayConfiguration.class, c -> {});
```

Comment:
Our mapper, argument, and collector registries might fit in this paradigm. :thinking: